src/sentry/conf/server.py

```diff
@@ -2444,6 +2449,8 @@ def build_cdc_postgres_init_db_volume(settings):
     KAFKA_INGEST_PERFORMANCE_METRICS: {"cluster": "default"},
     KAFKA_SNUBA_GENERIC_METRICS: {"cluster": "default"},
     KAFKA_INGEST_REPLAYS_RECORDINGS: {"cluster": "default"},
+    # Metrics Testing Topics
+    KAFKA_SNUBA_GENERICS_METRICS_DUMMY: {"cluster": "default"},
```

Comment:
Remember this will have to be overridden in getsentry in order to avoid hitting the default cluster which is ingest. 
This will be critical before we run the experimental indexer.